### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by fluxxu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ routerConfig : TransitRouter.Config Route Action Model
 routerConfig :
   { mountRoute : Route -> Route -> Model -> (Model, Effects Action)
   , getDurations : Route -> Route -> Model -> (Float, Float)
-  , actionWrapper : TransitRouter.Action -> Action
+  , actionWrapper : (TransitRouter.Action, Route) -> Action
```
Comment (etaque) [inline README.md]: Thanks for the PR! 
You left a comma here, also parenthesis are not necessary: `actionWrapper : TransitRouter.Action Route -> Action`

Comment (fluxxu) [inline README.md]: Fixed.  :D

**Comment by etaque** [comment]
Thanks, could you also remove the parenthesis? They are not necessary and people could think it's a tuple if they look up too fast :)

**Comment by fluxxu** [comment]
You are right. 
Done :)

**Comment by etaque** [comment]
:+1:

**Pull request merged**